addons/mail/static/src/components/thread_view/thread_view_tests.js

```diff
@@ -1367,6 +1367,80 @@ QUnit.test('[technical] message list with a full page of empty messages should s
     );
 });
 
+QUnit.test('new messages separator on receiving a message after having posted a command while composer is not focused [REQUIRE FOCUS]', async function (assert) {
+    // The goal of removing the focus is to ensure the thread is not marked as seen automatically.
+    // Indeed that would trigger channel_seen no matter what, which is already covered by other tests.
+    // The goal of this test is to cover the conditions specific to transient messages,
+    // and the conditions from focus would otherwise shadow them.
```

Comment:
It's not clear what's the spec of this test.

Don't think you have to mention channel commands in this test. I think just the concept of transient message is sufficient. Test can actually make one with channel command, but the spec should not be restricted to channel command (wrongly putting too much emphasis on this aspect).

At a glance, the test asserts good positioning of the new message separator with actual message and transient message next to each other. This should be clear just at a glance on test name.

Reply:
Indeed, command is just the flux, while the fix is generic to transient.